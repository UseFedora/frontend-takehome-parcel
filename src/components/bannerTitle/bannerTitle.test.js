import React from 'react';
import { shallow } from 'enzyme';
import BannerTitle from './';

describe('BannerTitle', () => {
  it('should render correctly', () => {
    const wrapper = shallow(
      <BannerTitle
        label="Test Banner Title"
      />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should output the supplied label', () => {
    const wrapper = shallow(
      <BannerTitle
        label="Test Banner Title"
      />
    );
    expect(wrapper.text()).toEqual('Test Banner Title');
  });
});
