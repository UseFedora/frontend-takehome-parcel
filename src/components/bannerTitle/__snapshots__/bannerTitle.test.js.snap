// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BannerTitle should render correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <BannerTitle
    label="Test Banner Title"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "Test Banner Title",
      "className": undefined,
    },
    "ref": null,
    "rendered": "Test Banner Title",
    "type": "h4",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Test Banner Title",
        "className": undefined,
      },
      "ref": null,
      "rendered": "Test Banner Title",
      "type": "h4",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
